ghost/admin/app/styles/layouts/main.css

```diff
@@ -1184,11 +1184,6 @@ body:not(.gh-body-fullscreen) .gh-viewport {
     max-width: var(--main-layout-content-maxwidth);
 }
 
-.gh-nav-hidden + .gh-main .gh-canvas {
```

Comment:
@sandeep-singhDEV 

This removes the padding in non-mobile sizes too which is not ideal. Instead of deleting please update your change with the following, using a media query for the appropriate screen size:

```
@media (min-width: 800px) {
    .gh-nav-hidden + .gh-main .gh-canvas {
        padding-left: 80px;
        padding-right: 80px;
    }
}
```